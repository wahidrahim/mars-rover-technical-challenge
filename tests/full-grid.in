1 1
0 0 N
MRM
0 0 N
M
0 0 E
MLM
0 0 N
MLMLMMRMLM
