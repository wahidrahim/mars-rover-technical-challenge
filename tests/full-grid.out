1 1 E
0 1 N
1 0 N
0 0 S
